### PR TITLE
new icon: solidjs (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by AbdelrahmanDwedar**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1253**

## Link to prove your SVG is correct and up-to-date.

https://www.solidjs.com/media

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Wordmark icons aren't centered and resized correctly. Look at the file previews

**Comment by lunatic-fox** [comment]
**This PR closes #1253 (staled PR)**

**Comment by AbdelrahmanDwedar** [comment]
Oh I didn't see it. Sorry.

**Comment by AbdelrahmanDwedar** [comment]
> Wordmark icons aren't centered and resized correctly. Look at the file previews

I'll check them out, maybe I forgot to do something.

**Comment by AbdelrahmanDwedar** [comment]
I don't know what happened in the plain wordmark version the colors seem to disappear in the image.

**Comment by lunatic-fox** [comment]
> Oh I didn't see it. Sorry.

No problem! 🙂

File: icons/solidjs/solidjs-plain-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...978.03766-3.12534-1.92039-4.8198-4.78214-4.8198h-2.52285l-.03765 9.63959zm15.87895 1.01668a3.86713 3.86713 0 0 0 2.65842-.9489 3.38891 3.38891 0 0 0 1.07692-2.67347v-7.94889h-2.35342V65.526a1.5815 1.5815 0 0 1-.37654 1.12964 1.35933 1.35933 0 0 1-1.04304.4029q-1.10704 0-1.61538-1.29908l-1.99193 1.1108q.97526 2.45132 3.64497 2.45132zm14.10542.02636c1.30661 0 2.33082-.32383 3.06508-.97525a3.24206 3.24206 0 0 0 1.09952-2.55675 2.66971 2.66971 0 0 0-.94137-2.13502c-.6326-.54222-1.64551-1.00914-3.03873-1.40828a4.42818 4.42818 0 0 1-1.41581-.59118.96772.96772 0 0 1-.4142-.80957 1.01667 1.01667 0 0 1 .4142-.84723 1.73211 1.73211 0 0 1 1.09575-.32006 2.13502 2.13502 0 0 1 2.15008 1.40075l1.94674-1.06563a3.78429 3.78429 0 0 0-1.53254-1.82248 4.58257 4.58257 0 0 0-2.45885-.6439 4.18343 4.18343 0 0 0-2.80526.97902 3.05379 3.05379 0 0 0-1.16353 2.46638 2.8241 2.8241 0 0 0 .87358 2.11243q.78322.79074 2.77515 1.38945a6.21678 6.21678 0 0 1 1.70199.68532.87735.87735 0 0 1 .44809.78698 1.03174 1.03174 0 0 1-.48575.86982 2.12372 2.12372 0 0 1-1.2426.34266 2.63582 2.63582 0 0 1-2.4852-1.5928l-2.03335 1.05057a4.08553 4.08553 0 0 0 1.69069 1.96933 5.17374 5.17374 0 0 0 2.75632.71544z" class="g"/></svg>
```
Comment (lunatic-fox) [inline icons/solidjs/solidjs-plain-wordmark.svg:1]: It seems that the bottom path of the "s" is missing.
Here's some suggestion of fixing made with Inkscape CLI from `original-wordmark` version.

### Preview
preview

> Note: even it's black, actually the path has no fill color, so it'll be filled by `color` property of `devicon.json` entry.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M40.25 67.017a4.374 4.374 0 0 0 2.952 1.24c1.239 0 2.26-.73 2.26-1.969 0-2.843-5.65-2.15-5.65-5.65 0-1.64 1.458-3.024 3.535-3.024a4.374 4.374 0 0 1 3.026 1.093l-.55.948a3.783 3.783 0 0 0-2.48-.984c-1.457 0-2.368.947-2.405 1.931 0 2.697 5.65 1.932 5.65 5.65 0 1.676-1.313 3.061-3.427 3.061a4.87 4.87 0 0 1-3.571-1.458zm17.895-9.33a5.733 5.733 0 0 1 5.759 5.795 5.832 5.832 0 0 1-5.759 5.904h-.036a5.832 5.832 0 0 1-5.76-5.904v-.037a5.733 5.733 0 0 1 5.796-5.758zm-.036 10.679a4.738 4.738 0 0 0 4.628-4.844v-.04c0-2.697-2.04-4.775-4.592-4.775a4.636 4.636 0 0 0-4.629 4.738c0 2.77 2.041 4.884 4.593 4.92zm12.355-10.46h1.094L71.52 68.22h5.249v.984H70.39zm12.356.036h1.094l-.037 11.335h-1.093zm8.456 0h3.645c3.39 0 5.685 2.077 5.685 5.686 0 3.608-2.296 5.649-5.685 5.649h-3.645zm3.499 10.35c2.77 0 4.629-1.64 4.665-4.664.037-3.026-1.859-4.666-4.629-4.666H92.37l-.036 9.33zm15.37.985a3.743 3.743 0 0 0 2.573-.919 3.28 3.28 0 0 0 1.042-2.587v-7.694h-2.278v7.526a1.53 1.53 0 0 1-.364 1.093 1.316 1.316 0 0 1-1.01.39q-1.071 0-1.563-1.257l-1.928 1.075q.944 2.373 3.528 2.373zm13.653.025c1.265 0 2.256-.313 2.967-.944a3.138 3.138 0 0 0 1.064-2.474 2.584 2.584 0 0 0-.911-2.067c-.612-.525-1.593-.977-2.941-1.363a4.286 4.286 0 0 1-1.37-.572.937.937 0 0 1-.402-.784.984.984 0 0 1 .401-.82 1.677 1.677 0 0 1 1.06-.31 2.067 2.067 0 0 1 2.082 1.356l1.884-1.031a3.663 3.663 0 0 0-1.483-1.765 4.436 4.436 0 0 0-2.38-.623 4.05 4.05 0 0 0-2.715.948 2.956 2.956 0 0 0-1.127 2.387 2.734 2.734 0 0 0 .846 2.045q.758.765 2.686 1.345a6.018 6.018 0 0 1 1.648.663.85.85 0 0 1 .433.762.999.999 0 0 1-.47.842 2.056 2.056 0 0 1-1.203.332 2.551 2.551 0 0 1-2.405-1.542l-1.968 1.017a3.955 3.955 0 0 0 1.636 1.906 5.008 5.008 0 0 0 2.668.692zm-104.36.609a7.705 7.705 0 0 0-8.212-2.606L.46 70.789s9.111 6.867 16.15 5.263l.62-.17c2.882-.916 3.866-3.61 2.208-5.97zm3.382-6.021a7.694 7.694 0 0 0-8.23-2.606L3.842 64.768.46 70.79l19.255 3.28 3.451-6.14c.69-1.178.62-2.661-.346-4.04zM8.646 56.123l-.624.175c-2.879.93-3.863 3.623-2.208 5.985a7.698 7.698 0 0 0 8.215 2.587l10.76-3.502s-9.076-6.849-16.143-5.245zm19.248.087s-9.094-6.866-16.15-5.263l-.62.172a4.501 4.501 0 0 0-2.398 1.684 4.075 4.075 0 0 0-.259.43L5.861 57.66l4.501.878a8.018 8.018 0 0 0 6.419 1.262l8.018 1.567z"/></svg>

```

File: icons/solidjs/solidjs-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...5c8.79198 5.5949 19.98177 7.9927 30.37229 5.5949l36.76644 7.19343z"/><path fill="url(#a)" d="M127.11625 33.79255S84.75491 2.621 51.98482 9.81444l-2.39781.79927c-4.79563 1.59854-8.79198 3.99635-11.1898 7.19343l-1.59853 2.39781L24.80962 40.986l20.78103 3.99635c8.79198 5.5949 19.98177 7.9927 30.37229 5.5949l36.76644 7.19343z" opacity=".3"/><path fill="#518ac8" d="m38.39722 33.79255-3.19708.79927C21.61254 38.58818 17.61619 51.3765 24.80962 62.5663c7.9927 10.39052 24.77739 15.98541 38.36499 11.98906l49.55477-16.78468S70.36804 26.59912 38.39722 33.79255z"/><path fill="url(#b)" d="m38.39722 33.79255-3.19708.79927C21.61254 38.58818 17.61619 51.3765 24.80962 62.5663c7.9927 10.39052 24.77739 15.98541 38.36499 11.98906l49.55477-16.78468S70.36804 26.59912 38.39722 33.79255z" opacity=".3"/><path fill="url(#c)" d="M103.93741 69.75973a35.96717 35.96717 0 0 0-38.365-11.98906L16.01765 73.75608.03224 101.73055l89.5183 15.18614 15.9854-28.77374c3.19709-5.5949 2.39782-11.98906-1.59853-18.38322z"/><path fill="url(#d)" d="M87.952 97.7342a35.96717 35.96717 0 0 0-38.365-11.98906L.03224 101.73055s42.36134 31.97082 75.13143 23.97812l2.39782-.79927c13.5876-3.99636 18.38322-16.78469 10.39051-27.1752Z"/></svg>
```
Comment (lunatic-fox) [inline icons/solidjs/solidjs-original.svg:1]: Seems that this icon is not well aligned to the center. 🤔

File: icons/solidjs/solidjs-plain.svg [hunk 3]
```diff
@@ -0,0 +1 @@
... 4.00563c8.81238 5.60789 20.02815 8.01126 30.44278 5.60789l36.8518 7.21013z"/><path fill="url(#a)" d="M127.6114 30.308S85.15173-.93593 52.30556 6.2742l-2.40337.80113c-4.80676 1.60225-8.81239 4.00563-11.21577 7.21013l-1.60225 2.40338-12.01689 20.82927 20.82928 4.00563c8.81238 5.60789 20.02815 8.01126 30.44278 5.60789l36.8518 7.21013z" opacity=".3"/><path fill="#518ac8" d="m38.68642 30.308-3.2045.80112c-13.61914 4.00563-17.62477 16.82364-10.41464 28.0394 8.01126 10.41465 24.8349 16.02253 38.45405 12.0169l49.66981-16.82365S70.73146 23.09786 38.68642 30.308z"/><path fill="url(#b)" d="m38.68642 30.308-3.2045.80112c-13.61914 4.00563-17.62477 16.82364-10.41464 28.0394 8.01126 10.41465 24.8349 16.02253 38.45405 12.0169l49.66981-16.82365S70.73146 23.09786 38.68642 30.308z" opacity=".3"/><path fill="url(#c)" d="M104.37875 66.35866a36.05067 36.05067 0 0 0-38.45404-12.01689L16.25489 70.3643.23237 98.4037l89.72612 15.2214L105.981 84.78456c3.2045-5.60788 2.40337-12.0169-1.60226-18.4259z"/><path fill="url(#d)" d="M88.35623 94.39807A36.05067 36.05067 0 0 0 49.9022 82.38118L.23237 98.4037s42.45968 32.04505 75.30585 24.03379l2.40338-.80112c13.61914-4.00564 18.4259-16.82366 10.41463-27.2383z"/></svg>
```
Comment (lunatic-fox) [inline icons/solidjs/solidjs-plain.svg:1]: Seems that this icon also is not well aligned to the center and the paths are not unified into one.
If you need some help in unifying the paths on Inkscape, take a look at this [comment](https://github.com/devicons/devicon/issues/1306#issuecomment-1296052892).
And if you want some help about alignment on Inkscape, just keep reading.

### Align to the center of the page
> **Steps to follow if you're editing some `original` or `original-wordmark` version that has more than one path will be marked with `*`**
- Open the "Align and Distribute" tab by going to `Object > Align and Distribute...` or by key command `Shift + Ctrl + A`
- \* Select all `Ctrl + A` and group it `Ctrl + G`
- On `Align`, select `Relative to: Page`
- Select to center to the vertical and horizontal axis
- \* Select all `Ctrl + A` and ungroup it `Shift + Ctrl + G`

File: devicon.json [hunk 4]
```diff
@@ -7361,6 +7361,36 @@
...
+            {
+                "base": "original",
+                "alias": "plain"
+            },
+            {
+                "base": "original-wordmark",
+                "alias": "plain-wordmark"
+            }
+        ]
+	},
```
Comment (lunatic-fox) [inline devicon.json:7393]: Fix alignment and remove unnecessary aliases. 🙂
```suggestion
    {
        "name": "solidjs",
        "altnames": [],
        "tags": [
            "javascript",
            "framework",
            "frontend"
        ],
        "versions": {
          "svg": [
              "original",
              "original-wordmark",
              "plain",
              "plain-wordmark"
          ],
          "font": [
              "plain",
              "plain-wordmark"
          ]
        },
        "color": "#518ac8",
        "aliases": []
    },
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]

**Review by lunatic-fox — DISMISSED** [review]
Icons are ok now and conflicts are resolved in `devicon.json`
Thank you for this contribution! 🚀

File: devicon.json [hunk 5]
```diff
@@ -8059,6 +8059,29 @@
...
+              "original-wordmark",
+              "plain",
+              "plain-wordmark"
+          ],
+          "font": [
+              "plain",
+              "plain-wordmark"
+          ]
+        },
+        "color": "#518ac8",
```
Comment (Snailedlt) [inline devicon.json:8082]: Accord to https://www.solidjs.com/media the primary color should be `#2c4f7c`
```suggestion
        "color": "#2c4f7c",
```

File: icons/solidjs/solidjs-original-wordmark.svg [hunk 6]
```diff
@@ -0,0 +1 @@
...63.85.85 0 0 1 .433.762.999.999 0 0 1-.47.842 2.056 2.056 0 0 1-1.203.332 2.551 2.551 0 0 1-2.405-1.542l-1.968 1.017a3.955 3.955 0 0 0 1.636 1.906 5.008 5.008 0 0 0 2.668.692z"/><path fill="#76b3e1" d="M27.798 56.627s-9.166-6.745-16.258-5.189l-.519.173c-1.037.346-1.902.865-2.421 1.557l-.346.519-2.594 4.497 4.496.864c1.903 1.211 4.324 1.73 6.573 1.211l7.956 1.557z"/><path fill="url(#a)" d="M27.798 56.627s-9.166-6.745-16.258-5.189l-.519.173c-1.037.346-1.902.865-2.421 1.557l-.346.519-2.594 4.497 4.496.864c1.903 1.211 4.324 1.73 6.573 1.211l7.956 1.557z" opacity=".3"/><path fill="#518ac8" d="m8.6 56.627-.692.173c-2.94.865-3.805 3.632-2.248 6.054 1.73 2.248 5.361 3.459 8.302 2.594l10.723-3.632S15.518 55.07 8.6 56.627z"/><path fill="url(#b)" d="m8.6 56.627-.692.173c-2.94.865-3.805 3.632-2.248 6.054 1.73 2.248 5.361 3.459 8.302 2.594l10.723-3.632S15.518 55.07 8.6 56.627z" opacity=".3"/><path fill="url(#c)" d="M22.783 64.41a7.783 7.783 0 0 0-8.303-2.594L3.757 65.275l-3.46 6.054 19.372 3.286 3.46-6.227c.691-1.21.518-2.594-.346-3.978z"/><path fill="url(#d)" d="M19.323 70.464a7.783 7.783 0 0 0-8.302-2.595L.298 71.33s9.167 6.918 16.258 5.188l.519-.173c2.94-.864 3.978-3.632 2.248-5.88z"/></svg>
```
Comment (Snailedlt) [inline icons/solidjs/solidjs-original-wordmark.svg:1]: This should be replaced with the version from their media assets page: https://www.solidjs.com/media

**Review by Snailedlt — CHANGES_REQUESTED** [review]

**Review by Snailedlt — APPROVED** [review]
Looking good! Thank you for your contribution! ✔️

**Pull request merged**